plasmapy/examples/plot_physics.py

```diff
@@ -57,4 +57,79 @@
     formulary.plasma_frequency(electron_concentration))
 plt.xlabel("Electron Concentration (m^-3)")
 plt.ylabel("Langmuir Wave Plasma Frequency (rad/s)")
+plt.grid()
+plt.show()
+
+############################################################
+# Let's compare our data to real-world data of plasma devices
+
+# Our 'Data'
+print("---Our Data---")
+print("n_e: " + str(electron_concentration))
+print("T_e: " + str(electron_temperature))
+print("B: " + str(B))
+plt.scatter(electron_concentration, formulary.plasma_frequency(electron_concentration))
+
+# IRT1 Tokamak Data
+# http://article.sapub.org/pdf/10.5923.j.jnpp.20110101.03.pdf
+n_e = 1.2e19 / u.m**3
+T_e = 136.8323 * u.eV
+B = 0.82 * u.T
+print("---IRT1 Tokamak Data---")
+print("n_e: " + str(n_e))
+print("T_e: " + str(T_e))
+print("B: " + str(B))
```

Comment:
We could probably cut the prints, so there's less clutter in the code and in the output - just plotting them should be enough!